gemrb/plugins/SDLVideo/SDL20Video.cpp

```diff
@@ -126,14 +126,7 @@ int SDL20VideoDriver::UpdateRenderTarget(const Color* color)
 		return ret;
 	}
 
-	if (screenClip.Dimensions() == screenSize)
```

Comment:
we actually need this to silence complaints from the Metal backend. It too might just be an SDL bug for all I know, but debug builds of the renderer will assert without this under the current SDL 2.0.10 at least.

Reply:
Argh, ok

Reply:
I rephrase the comment in a way non-metal readers understand. :wink: